ui/src/views/Releases/ListReleases/index.jsx

```diff
@@ -533,18 +653,33 @@ function ListReleases(props) {
   };
 
   const handleViewScheduledChangeDiff = async release => {
-    const result = await fetchRelease(release.name);
-
-    setDrawerState({
-      ...drawerState,
-      item: {
-        firstRelease: result.data.data,
-        secondRelease: release.scheduledChange.data,
-        firstFilename: `Data Version: ${release.data_version}`,
-        secondFilename: 'Scheduled Change',
-      },
-      open: true,
-    });
+    if (release.api_version === 1) {
```

Comment:
In one of the `useEffect` methods in `ListRules/index.jsx` we make a request to the v2 endpoint and if there is an error, we fallback to v1. Would it make sense to check the version first before firing up the request?

Reply:
I believe you're talking about https://github.com/mozilla-releng/balrog/blob/012237085dee260f85da7192389088c82c41d445/ui/src/views/Rules/ListRules/index.jsx#L963? I could go either way on that -- checking the version first would require querying the list of releases for both API versions, which is data that is otherwise unused on that view.

Reply:
Should be okay the way it is right now.